srcpkgs/dtools/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'dtools'
+pkgname=dtools
+# keep this synchronized with libphobos and dmd
+version=2.081.1
+revision=1
+wrksrc="tools-${version}"
+hostmakedepends="dmd"
+makedepends="libcurl-devel"
+depends="libphobos>=${version} libcurl"
```

Comment:
the libcurl dependency should be auto-detected, you don't need to specify it.

Reply:
For future reference, how exactly does autodetection work?

Reply:
It checks which libraries the binaries link to and cross references them with the file `common/shlibs` (in this repo). Typically if you specify `libfoo-devel` in the makedepends you don't also need to put it in depends.

Reply:
Thanks for explaining!